.travis.yml

```diff
@@ -1,10 +1,9 @@
 sudo: false
 language: python
 python:
-- 2.7
-- 3.6
 - 3.7
 - 3.8
+- 3.9
```

Comment:
Kben eens bij bram gaan horen. Nieuwe servers zijn weliswaar 3.8, maar omdat er nog steeds bestaande servers zijn die 3.6 draaien zouden we hier best ook op 3.6 blijven testen nog een tijdje, zoals oe-utils.

In de praktijk draaien we 3.7 niet, dus als we travis load willen verkleinen zouden we wel 3.7 kunnen skippen.

Reply:
aie, rdflib vereist python 3,7+. Dan moet die terug downgegrade worden, tenzij python op de bestaande servers omhoog mag. 
Fyi, support voor python 3.6 eindigt op 23 Dec, dus misschien wel een goeie reden om dat uit te voeren ineens

Reply:
RDFlib 6.0.0 is een vrij grote change, die heeft ook ingebakken support voor JSON/LD. Ik zou die voor skosprovider_getty en skosprovider_rdf liever eens apart aangepakt zien worden, kan zijn dat dat nog meer gevolgen zal hebben.

Dus, nu liefst houden op de laatste versie in de 5.x.x reeks en compatibiliteit met Python 3.6 behouden.